<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Текстовый предиктор</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLoadTrain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>340</y>
      <width>491</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить текст для обучения предиктора</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>371</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Вводите текст для предиктора</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>20</y>
      <width>256</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>0</y>
      <width>241</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Возможные продолжения</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="editTrain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>491</width>
      <height>151</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>481</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Загруженный в предиктор обучающий текст</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClearUser">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>140</y>
      <width>106</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="editUser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>491</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
